src/components/App.tsx

```diff
@@ -900,6 +904,13 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         gridSize: this.props.gridModeEnabled ? GRID_SIZE : null,
       });
     }
+
+    if (prevProps.name !== this.props.name && this.props.name) {
```

Comment:
```suggestion
    if (prevProps.name !== this.props.name) {
```
This should be enough to compare..

Reply:
`Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.`